src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy

```diff
@@ -231,6 +235,10 @@ class DockerCreateContainer extends AbstractDockerRemoteApiTask {
         restartPolicy = "${name}:${maximumRetryCount}"
     }
 
+    void pid(String name) {
+        pid = "${name}"
+    }
```

Comment:
We generally don't use methods unless there is a good reason. What's the thinking here?

Reply:
Let's remove this method. Groovy automatically generates a setter method.

Reply:
Oh good to know. Yes I will remove that method then. Should I push it as a 2nd commit or should I commit locally, squash both commits and force push it into the pr/651 branch so that it remains one commit/push on pr/651? 

And I don't get why the Travis CI build failed.
It says "Caused by: org.gradle.api.GradleException: The whole test suite should be executed. The test com.bmuschko.gradle.docker.DockerSpringBootApplicationPluginFunctionalTest.Can create image for a Spring Boot application and push it to DockerHub [#plugin.identifier plugin] was skipped. Please check the execution condition!" .. do I have any influence on that when pushing?

Reply:
Ignore the Travis CI failure for now as it's a known issue. I would send in another commit with the fix.